tests/local.c

```diff
@@ -7,5 +12,10 @@ int main(void)
     char cc = 'a';
     char *ptr = &cc;
     printf("%c\n", *ptr);
+    struct Value a;
```

Comment:
Shall we add new entries about pointers to structures for `struct` test item?

Reply:
Like put into the `tests/struct.c`?

Reply:
It would be better if all structure related test entries are landed in file `tests/struct.c`